README.md

```diff
@@ -1,219 +1,41 @@
-    - `setUp` finds no running PocketIC server instance, so it launches one and discovers its port
-    - The `PocketIC()` constructor requests a new instance from the PocketIC server
-    - The server returns an instance id to the constructor, which completes and binds to `self.pic`
-    - `setUp` completes
-- `test_one` completes
-- `test_two` invokes `setUp`
-    - `setUp` finds a running PocketIC server instance and discovers its port
-    - ... (continues like above, overwriting all `self.*` fields, and using a new, independent IC instance)
-- etc.
-- After the PocketIC server is idle for a while (30s), it shuts down.
-
-### Using the canister interface 
-
-Using the IC interface to create and call canisters is familiar to canister developers and resembles the real IC interface. 
-
-However, due to Python's ease of reflection and the work of [RocketLab](https://github.com/rocklabs-io/ic-py), we offer a more convenient way to interact with your canisters. For this feature to work, you need a candid file which describes your canister's interface. 
+With PocketIC, you can test your canisters with just a few lines of Python code:
 
 ```python
 from pocket_ic import PocketIC
 
-WASM_PATH = "./counter.wasm"
-CANDID_PATH = "./counter.did"
-
-candid = open(CANDID_PATH, "r", encoding="utf-8").read()
-wasm = open(WASM_PATH, "r", encoding="utf-8").read()
+pic = PocketIC()
+my_canister = pic.create_and_install_canister_with_candid(...)
 
-# Some canisters require init_args for code installation. 
-# These need to be candid encoded. See examples/ledger_canister/.
-# For the counter canister, we can do without.
-init_args = dict()
-
-ic = PocketIC()
-# Create a canister obj with the provided candid interface.
-counter_canister = ic.create_and_install_canister_with_candid(candid, wasm, init_args)  
-# The canister's candid specifies and `inc` method with no arguments, so we can just call it:
-counter_canister.inc()  
-# Similarly for a function called `read`, which returns a Nat:
-assert(counter_canister.read() == 1) 
+# call your canister functions with native Python syntax
+respone = my_canister.some_canister_function()
+assert(response == 'Hello, World!')
 ```
 
-If you need help with candid-encoding your `init_args`, canister call arguments and responses, check out the community developed [Python-agent](https://github.com/rocklabs-io/ic-py), which offers some useful candid functionality.
+## Getting Started
 
+### Quickstart
+* Download the **PocketIC binary** for [Linux](https://download.dfinity.systems/ic/71adba179679a1090baa26cadcceadec311f57b3/openssl-static-binaries/x86_64-linux/pocket-ic.gz) or [macOS](https://download.dfinity.systems/ic/71adba179679a1090baa26cadcceadec311f57b3/openssl-static-binaries/x86_64-darwin/pocket-ic.gz).
+* Make sure the binary is available through your `$PATH` variable; you can verify that everything works by calling `pocket-ic --help` from your terminal.
+* Run `pip3 install pocketic-py` to get the Python library. 
```

Comment:
maybe add: "from your virtual environment"? 

I don't like suggesting a bad practice. We don't know the expertise of our audience.